### PR TITLE
{phys}[dummy] COMSOL v5.2a

### PR DESCRIPTION
**Pull request by Micket**
Depends on easybuilders/easybuild-easyblocks#1296

### CODE REVIEW

File: easybuild/easyconfigs/c/COMSOL/COMSOL-5.2a.eb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Applications"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCELOWER_TAR_GZ]
+
+# License is required (EB_COMSOL_LICENSE), e.g:
+# EB_COMSOL_LICENSE="FFFFFFFFF-BBB-123456-123456"
+# EB_COMSOL_LICENSE="1718@licsvr1 1718@licsvr2"
+# EB_COMSOL_LICENSE="/path/to/license.dat"
```
Comment (boegel) [inline easybuild/easyconfigs/c/COMSOL/COMSOL-5.2a.eb]: This should be updated according to the changes in https://github.com/easybuilders/easybuild-easyblocks/pull/1317?

File: easybuild/easyconfigs/c/COMSOL/COMSOL-5.2a.eb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name = 'COMSOL'
+version = '5.2a'
+
+homepage = 'https://www.comsol.se/'
+description = """COMSOL Multiphysics - Simulation Tool for Electrical, Mechanical, Fluid Flow, and Chemical
+Applications"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (boegel) [inline easybuild/easyconfigs/c/COMSOL/COMSOL-5.2a.eb]: I think it's worth including a comment on how to obtain this tarball?

I think COMSOL typically comes as a `.iso`? Is it clear which tarball to grab from that, or should we specify it more clearly?

Also, please include a SHA256 checksum?

**Comment by migueldiascosta** [comment]
@Micket ping?

**Comment by Micket** [comment]
This is outdated by several new comsol releases already, so probably not very interesting anymore.

**Pull request closed without merging**